docs/docs/schema/pipeline.json

```diff
@@ -204,42 +244,51 @@
         },
         "RepoAuthFlags": {
             "title": "RepoAuthFlags",
+            "description": "Authorisation-related flags for `helm repo`",
             "type": "object",
             "properties": {
                 "username": {
                     "title": "Username",
+                    "description": "Username",
                     "type": "string"
                 },
                 "password": {
                     "title": "Password",
+                    "description": "Password",
                     "type": "string"
                 },
                 "caFile": {
                     "title": "Cafile",
+                    "description": "Certificate file",
                     "type": "string",
                     "format": "path"
                 },
                 "insecureSkipTlsVerify": {
                     "title": "Insecureskiptlsverify",
+                    "description": "If true, Kubernetes API server's certificate will not be checked for validity\n        ",
```

Comment:
nitpick: would it be possible to strip trailing whitespace (newlines and spaces)?

Reply:
tbh I have no idea why they don't get stripped

Reply:
perhaps we should trim all linebreaks in general and let the the editor take care of it. wdyt?

Reply:
we can also do it in a followup, perhaps it's better to see a diff on this

Reply:
I agree. I managed to remove the newlines characters, but I can't seem to remove the whitespaces..

Reply:
> perhaps we should trim all linebreaks in general and let the the editor take care of it. wdyt?

I definitely agree